frontend/src/components/MultiplierSelector.jsx

```diff
@@ -5,25 +5,24 @@ import './multiplier.css';
 
 const MultiplierSelector = ({ setSelectedMultiplier, selectedToken }) => {
   const { data, isLoading, error } = useMaxMultiplier();
-  const [actualValue, setActualValue] = useState(1.0);
+  const [actualValue, setActualValue] = useState(0.0);
   const sliderRef = useRef(null);
   const isDragging = useRef(false);
-
-  const maxMultiplier = useMemo(() => {
-  return Math.round(parseFloat((data?.[selectedToken]))) || 5.0;
-}, [data, selectedToken]);
+  const maxMultiplier = useMemo(() => 10, [data, selectedToken]);
```

Comment:
it should be not hardcoded to 10, rather it should be equal to a value returned from a backend